core-kotlin-modules/core-kotlin-design-patterns/src/test/kotlin/com/baeldung/composite/MovieKodeinUnitTestUnitTest.kt

```diff
@@ -3,7 +3,7 @@ package com.baeldung.composite
 import org.junit.jupiter.api.Assertions.assertEquals
 import org.junit.jupiter.api.Test
 
-class MovieApplicationUnitTest {
+class MovieKodeinUnitTestUnitTest {
```

Comment:
Let's make sure we aren't changing this file in this PR - let's undo the rename of the class and file